core/src/test/kotlin/ch/ergon/dope/buildTest/AggregateFunctionsTest.kt

```diff
@@ -1,5 +1,6 @@
-package ch.ergon.dope
+package ch.ergon.dope.buildTest
```

Comment:
Not quite sure about this name, maybe we can find one that fits better?

Reply:
Do you have some ideas for names? I can't think of one.

Reply:
Talked about it in pairing